gravitee-apim-perf/scripts/run.sh

```diff
@@ -117,7 +117,7 @@ echo "Current K6 Context:"
 env | grep -e "^K6_.*=.*" || true
 
 ### If verbose mode enabled, then prepare K6 options.
-VERBOSE_OPTIONS=""
+VERBOSE_OPTIONS=${VERBOSE_OPTIONS:-false}
```

Comment:
is `false` interpreted as an empty string when using it in line 143 ?

Reply:
sorry, big mistake on my side. :( 
I fixed it